src/framed/std/core.clj

```diff
@@ -16,6 +16,14 @@
       (concat (f (first coll))
               (mapcat f (rest coll))))))
 
+(defn shuffle
+  "Same as clojure.core/shuffle but accepts source of randomness
+   for deterministic testing"
+  [^java.util.Random rng ^java.util.Collection coll]
```

Comment:
I believe so; all those that can be shuffled evidently, as this is copy-pasted from clojure.core. http://clojure.org/data_structures appears to confirm: `In addition, the collections:.....Implement the non-optional (read-only) portion of java.util.Collection`